### PR TITLE
Tpg fleet 181123

### PR DESCRIPTION
**Pull request by amathieson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ If you have a question or an idea, you can create an issue. Pull requests are we
 
 ## Authors
 - **Nicolas Ettlin** ([@Nicolapps](https://github.com/Nicolapps))
+- **Adam Mathieson** ([@AMathieson](https://github.com/amathieson)) - Help in transition to alternative API & Updates to vehicles
```
Comment (Nicolapps) [inline README.md:43]: Thank you for the change, you definitely deserve credit with all the help you’ve been providing!

File: tpgdata/vehicules/vehicules.php [hunk 2]
```diff
@@ -390,16 +390,16 @@ public function __construct(string $id) {
...
             $this->img_author = 'TramBusAl';
-            $this->img_link = 'https://twitter.com/trambusalOff';
+            $this->img_link = 'https://flickr.com/photos/trambusal/';
 
         } elseif (2003 <= $id && $id <= 2010) { // Sprinter
 
             $this->type = "Minibus";
             $this->name = "Mercedes-Benz Sprinter City 65";
-            $this->icon = 'notfound';
+            $this->icon = 'altas';
```
Comment (Nicolapps) [inline tpgdata/vehicules/vehicules.php:399]: This icon seems to be missing, did you mean to add a new one?

**Review by Nicolapps — CHANGES_REQUESTED** [review]
Thank you again for the update!

File: tpgdata/vehicules/vehicules.php (hunk 2 above)
Comment (amathieson) [reply tpgdata/vehicules/vehicules.php:399]: oops, my fault forgot to add it to the repo 🤦

**Review by Nicolapps — APPROVED** [review]

**Pull request merged**